services/discordService.js

```diff
@@ -8,21 +8,13 @@ const getDiscordMembers = async () => {
     expiresIn: config.get("rdsServerlessBot.ttl"),
   });
 
-  try {
-    const response = await fetch(`${DISCORD_BASE_URL}/discord-members`, {
+  const response = await (
```

Comment:
- Is the error case being handled by the consumer of this service?

Reply:
yes, the consumer is the controller named syncExternalAccounts which has a try-catch block to handle errors.